frontend/src/components/UserUpdateSfdsAuth.vue

```diff
@@ -0,0 +1,283 @@
+                          outlined
+                          dense
+                          multiple
+                          chips
+                      ></v-autocomplete>
+                    </v-col>
+                    <v-col class="col-4">
+                      <label class="required" for="sfds-permissions" >Permission</label>
+                      <v-select
+                          id="sfds-permissions"
+                          v-model="currentSfdsAuthorization.p"
+                          :items="sfdsPermissions"
+                          required
+                          :rules="[v => !!v || 'A permission is required']"
+                          outlined
+                          dense
+                      ></v-select>
+                    </v-col>
+                  </v-row>
+                </v-form>
+              </v-container>
+            </v-card-text>
+            <v-card-actions>
+              <v-btn id="save-sfds-auth-btn" class="primary" @click="saveSfdsAuthorization">
+                Save
+              </v-btn>
+              <v-btn outlined class="primary--text" @click="closeDialog">
+                Cancel
+              </v-btn>
+            </v-card-actions>
+          </v-card>
+        </v-dialog>
+        <v-dialog v-model="deleteDialog" max-width="650px" >
+          <v-card >
+            <v-card-title class="headline">Are you sure you want to delete this authorization?</v-card-title>
+            <v-card-text class="black--text">
+              <v-alert id="sfds-delete-auth-alert" v-model="alertStatus" :type="alertType" dismissible>{{ alertMessage }}</v-alert>
+              <strong>Mailbox:</strong> {{ currentSfdsAuthorization.m }} <br/>
+              <strong>Use:</strong> {{ currentSfdsAuthorization.u }} <br/>
+              <strong>Permission:</strong> {{ currentSfdsAuthorization.p }}
+            </v-card-text>
+            <v-card-actions>
+              <v-btn id="confirm-delete-sfds-btn" class="red white--text" @click="deleteItemConfirm">Delete</v-btn>
+              <v-btn outlined class="primary--text" @click="closeDialog">Cancel</v-btn>
+            </v-card-actions>
+          </v-card>
+        </v-dialog>
+      </v-toolbar>
+    </template>
+    <template v-slot:item.actions="{ item }">
+      <v-icon
+          small
+          class="mr-2"
+          @click="editItem(item)">
+        mdi-pencil
+      </v-icon>
+      <v-icon
+          :id="item.m+'-delete-btn'"
+          small
+          @click="deleteItem(item)">
+        mdi-delete
+      </v-icon>
+    </template>
+  </v-data-table>
+</template>
+
+<script>
+import UsersRepository from "@/api/UsersRepository";
+
+export default {
+  name: "UserUpdateSfdsAuth",
+  data() {
+    return {
+      sfdsTableHeaders: [
+        { text: 'Mailbox', value: 'm' },
+        { text: 'Uses', value: 'u' },
+        { text: 'Permission', value: 'p' },
```

Comment:
short hand id's for the sfds object to save space in Keycloak's limited attribute size fields